frontend/src/components/Client/Dropdown/Dropdown.js

```diff
@@ -1,21 +1,66 @@
-const Dropdown = ({prefix, sortOptions = [], handleOptions, defaultValue}) => {
-  const options = sortOptions.map((option, i) => (
-    <option value={option.value} key={i}>{option.display}</option>
-  ))
+import Select from 'react-select'
+import { components } from 'react-select'
+
+const Dropdown = ({ prefix, options, handleOptions, defaultValue }) => {
```

Comment:
is it possible to set a default value here for prefix? You can probably set it to `""`, that way you can remove all the conditionals and just have it display nothing if defaulted